# Allow gnome-vfs-2.24 to build with gnutls >= 2.7.0
# Use pkg-config in place of gnutls own macro since it's not present anymore.
--- a/configure.in	2009-03-07 19:59:53.805507753 +0100
+++ b/configure.in	2009-03-07 18:34:36.928169018 +0100
@@ -686,14 +686,14 @@
     AC_MSG_ERROR([*** Can't use both openssl and gnutls at the same time. Please pick one only. ***])
   else
     AC_CHECK_HEADER(gcrypt.h,, AC_MSG_ERROR([*** Need gcrypt.h to compile with GnuTLS support ***]))
-    AM_PATH_LIBGNUTLS(1.0.0, [AC_DEFINE(HAVE_GNUTLS, 1, [Define to 1 if GnuTLS is available])
-			      have_ssl=true])
+    PKG_CHECK_MODULES(LIBGNUTLS, gnutls >= 1.0.0, [AC_DEFINE(HAVE_GNUTLS, 1, [Define to 1 if GnuTLS is available])
+    				 	   	  have_ssl=true])
 
     if test "x${LIBGNUTLS_LIBS}" = "x"; then
-      AM_PATH_LIBGNUTLS(0.5.1, [AC_DEFINE(HAVE_GNUTLS, 1, [Define to 1 if GnuTLS is available])
-                                 AC_DEFINE(GNUTLS_COMPAT, 1, [FIXME])
-				 have_ssl=true],
-                                 AC_MSG_ERROR([Unable to find GNUTLS]))
+      PKG_CHECK_MODULES(LIBGNUTLS, gnutls >= 0.5.1, [AC_DEFINE(HAVE_GNUTLS, 1, [Define to 1 if GnuTLS is available])
+      				   	     	    AC_DEFINE(GNUTLS_COMPAT, 1, [FIXME])
+						    have_ssl=true],
+						    AC_MSG_ERROR([Unable to find GNUTLS]))
     fi
   fi
 fi
